Source/OpenTK/Platform/Windows/WglHelper.cs

```diff
@@ -107,24 +107,95 @@ object SyncRoot
 
         IntPtr GetAddress(string function_string)
         {
-            IntPtr address = Wgl.GetProcAddress(function_string);
-            if (!IsValid(address))
+            if (IsGL11(function_string))
             {
-                address = Functions.GetProcAddress(WinFactory.OpenGLHandle, function_string);
+                return Functions.GetProcAddress(WinFactory.OpenGLHandle, function_string);
+            }
+            else
+            {
+                return Wgl.GetProcAddress(function_string);
             }
-            return address;
         }
 
         #endregion
 
         #region Private Members
 
-        static bool IsValid(IntPtr address)
+        static bool IsGL11(string function_string)
         {
-            // See https://www.opengl.org/wiki/Load_OpenGL_Functions
-            long a = address.ToInt64();
-            bool is_valid = (a < -1) || (a > 3);
```

Comment:
Isn't there value in keeping the -1..3 test, and only if it returns valid compare against the gl11 names?